crates/types/src/vote.rs

```diff
@@ -156,7 +177,7 @@ impl<TYPES: NodeType, VOTE: Vote<TYPES>, CERT: Certificate<TYPES, Voteable = VOT
 
         // TODO: Get the stake from the stake table entry.
         *total_stake_casted += stake_table_entry.stake();
-        total_vote_map.insert(key, (vote.signature(), vote.date_commitment()));
+        total_vote_map.insert(key, (vote.signature(), vote.vote_commitment()));
```

Comment:
I think now that we are calculating a commitment on this call we should not be calling `vote_commitment` repeatedly and just reuse the value already calculated above